bedrock/base/templates/macros.html

```diff
@@ -264,24 +264,18 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
 
     {# utm_campaign is used to identify specific marketing campaigns. (For example: trailhead or skyline) If there is one, the campaign name is prepended to the default value of `fxa-embedded-form`. #}
     {% if utm_campaign %}
-      <input type="hidden" name="utm_campaign" value="{{ utm_campaign }}-fxa-embedded-form" id="fxa-email-form-utm-campaign" />
+      <input type="hidden" name="utm_campaign" value="{{ utm_campaign }}-fxa-embedded-form" />
     {% else %}
-      <input type="hidden" name="utm_campaign" value="fxa-embedded-form" id="fxa-email-form-utm-campaign" />
+      <input type="hidden" name="utm_campaign" value="fxa-embedded-form" />
     {% endif %}
 
     {# utm_content is optional. It should only be declared when there is new messaging associated with the form. (For example: `get-the-rest-of-firefox`) #}
-    {% if utm_content %}
-      <input type="hidden" name="utm_content" value="{{ utm_content }}" id="fxa-email-form-utm-content" />
-    {% endif %}
+    <input type="hidden" name="utm_content" value="{{ utm_content }}" id="fxa-email-form-utm-content"/>
```

Comment:
If these are no longer included conditionally, should they have default values?

Reply:
I don't think they should have default values. if `value=""` it does not get passed and it is not a required value.

Reply:
Ok as long as it doesn't get passed that should keep things nice and simple 👍

Reply:
It looks like loading the page without any utm params in the URL results in a 400 response from the metrics request:

```
GET https://latest.dev.lcip.org/metrics-flow?form_type=email&entrypoint=mozilla.org-firefox-accounts&utm_source=mozilla.org-firefox-accounts&utm_campaign=trailhead-fxa-embedded-form&utm_content=&utm_term= 400
```

The value for `utm_content` also seems to be defaulting

Reply:
Oh shoot, you are right. Reaching out to Saptarshi for a recommendation on what a default value should be.

Reply:
oh shoot, you are right. I spoke with Saptarshi and Justin about a potential default `utm_content` value and Justin asked why we don't insert the hidden `utm_content` input if the value is present on the URL. I thought that could potentially be a good solution-

if `utm_content` exists on URL: check that input field exists > If exists: overwrite // If does not exist: insert one.